Form/Psr7RequestHandler.php

```diff
@@ -0,0 +1,188 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\PsrHttpMessage\Form;
+
+use Psr\Http\Message\ServerRequestInterface as Request;
```

Comment:
I would not alias `ServerRequestInterface`. In the context of a Symfony package, I would associate a `Request` with a HttpFoundation request, which makes this alias especially confusing.

Apart from that, do we really need an instance of `ServerRequestInterface` or would an instance of `RequestInterface` be enough?

Reply:
I'll modify the alias.

We do need the `ServerRequestInterface` to process the uploaded files.

Reply:
So, we could accept a `RequestInterface` for forms that don't have file uploads?

Reply:
I'm not sure that'd make sense @derrabus: this class deals with forms, which are server-side request processing.

Reply:
Agreed.